javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -23,33 +23,41 @@ function AppNavbar() {
         <Navbar.Brand data-testid="brand">Mapache Search</Navbar.Brand>
       </LinkContainer>
       <Nav>
-          { isAdmin &&
-              <NavDropdown title="Admin">
-                  <NavDropdown.Item href="/admin">Maintain Admins</NavDropdown.Item>
-                  <NavDropdown.Item href="/admin/slackUsers">Slack Users</NavDropdown.Item>
-                  <NavDropdown.Item href="/admin/teams">Manage Teams</NavDropdown.Item>
-                  <NavDropdown.Item href="/admin/searchInfo">Search Information</NavDropdown.Item>
-              </NavDropdown>
-          }
-          { (isMember || isAdmin)  &&
-              <NavDropdown title="Channels">
-                  <NavDropdown.Item href="/member/channels">List Channels</NavDropdown.Item>
-              </NavDropdown>
-          }
-          { (isMember || isAdmin)  &&
-              <NavDropdown title="Search">
-                  <NavDropdown.Item href="/member/messages/search">Message Search</NavDropdown.Item>
-              </NavDropdown>
-          }
+
+        {isAdmin &&
+          <NavDropdown title="Admin">
+            <NavDropdown.Item href="/admin">Maintain Admins</NavDropdown.Item>
+            <NavDropdown.Item href="/admin/slackUsers">Slack Users</NavDropdown.Item>
+            <NavDropdown.Item href="/admin/teams">Manage Teams</NavDropdown.Item>
+            <NavDropdown.Item href="/admin/searchInfo">Search Information</NavDropdown.Item>
+          </NavDropdown>
+        }
+        {(isMember || isAdmin) &&
+          <NavDropdown title="Channels">
+            <NavDropdown.Item href="/member/channels">List Channels</NavDropdown.Item>
+          </NavDropdown>
+        }
+        {(isMember || isAdmin) &&
+          <NavDropdown title="Search">
+            <NavDropdown.Item href="/member/messages/search">Message Search</NavDropdown.Item>
+          </NavDropdown>
+        }
+
         <LinkContainer to={"/about"}>
-            <Nav.Link>About</Nav.Link>
+          <Nav.Link>About</Nav.Link>
         </LinkContainer>
         <ProfileNav />
-          { (isMember || isAdmin) && 
-              <LinkContainer to={"/member/search"}>
-                  <Nav.Link>Search</Nav.Link>
-              </LinkContainer> 
-          }
+        {(isMember || isAdmin) && <NavDropdown title="Analyze Message Data">
```

Comment:
a nitpick, but to maintain consistency with the rest of the document, can you put the `NavDropdown` on a separate line?